[CLI] [doc] Add 'ti repl' command for Taichi REPL shell (#1765)
taichi-dev/taichi: docs/cli_utilities.rst

Examples
--------
Taichi provides a set of bundled examples.
You could run ``ti example -h`` to print the help message and get a list of available example names.
For instance, to run the basic ``fractal`` example, try: ``ti example fractal`` from your shell. (``ti example fractal.py`` should also work)

You may print the source code of example by running ``ti example -p fractal``, or ``ti example -P fractal`` for print with syntax highlight.
You may also save the example to current work directory by running ``ti example -s fractal``.

> 

Reviewer: Here do we want to have seperated lines? ;) maybe additional blank lines are essential if so?

Author: ```suggestion
Taichi provides a set of bundled examples.
You could run ``ti example -h`` to print the help message and get a list of available example names.
For instance, to run the basic ``fractal`` example, try: ``ti example fractal`` from your shell. (``ti example fractal.py`` should also work)

You may print the source code of example by running ``ti example -p fractal``, or ``ti example -P fractal`` for print with syntax highlight.

You may also save the example to current work directory by running ``ti example -s fractal``.

```

Changelog
---------
Sometimes it's convenient to view the changelog of the current version of Taichi.

> To do so from your shell, you could run ``ti changelog``.

Reviewer: ```suggestion
To do so, you could run ``ti changelog`` in your shell.
```
Maybe this ordering can make our expression more clear

REPL Shell
----------
Sometimes it's convenient to start a Python shell with ``import taichi as ti``

> as pre-loaded module for fast testing and confirmation.

Reviewer: ```suggsetion
as a pre-loaded module for fast testing and confirmation.
```

To do so from your shell, you could run ``ti repl``.

System information
------------------

> When reporting bugs, consider run ``ti diagnose`` and attach its output,
> this helps maintainers to see more about your system information so that
> they could hopefully resolve the issue easier.

Reviewer: ```suggestion
When you try to report potential bugs in an issue, please consider running ``ti diagnose`` and offer its output as an attachment.
This can help maintainers to see more about your system information so that
they could hopefully resolve the issue easier.
```

You may print the source code of example by running ``ti example -p fractal``, or ``ti example -P fractal`` for print with syntax highlight.

You may also save the example to current work directory by running ``ti example -s fractal``.

Changelog
---------
Sometimes it's convenient to view the changelog of the current version of Taichi.
To do so, you could run ``ti changelog`` in your shell.

REPL Shell
----------
Sometimes it's convenient to start a Python shell with ``import taichi as ti``
as a pre-loaded module for fast testing and confirmation.
To do so from your shell, you could run ``ti repl``.

System information
------------------
When you try to report potential bugs in an issue, please consider running ``ti diagnose`` and offer its output as an attachment.
This can help maintainers to see more about your system information so that
they could hopefully resolve the issue easier.

.. _cli_video_tools:

Converting PNGs to video
------------------------
Sometimes it's convenient to convert a series of ``png`` files into a single
video when showing your result to others.

> For example, you have ``000000.png``, ``000001.png``, ... generated according
> to :ref:`export_results` in **current working directory**.

Reviewer 2: ```suggestion
For example, suppose you have ``000000.png``, ``000001.png``, ... generated according
to :ref:`export_results` in the **current working directory**.
```
